data/json/items/tools.json

```diff
@@ -3414,7 +3414,15 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "weapons",
-    "use_action": "MOLOTOV"
+    "use_action": {
+        "type": "transform",
+        "target": "MOLOTOV_LIT",
```

Comment:
I think it's better to write it in the same manner as target item id (in lowercase : ```molotov_lit```).